protos/sigstore_common.proto

```diff
@@ -158,15 +158,22 @@ message X509Certificate {
 }
 
 enum SubjectAlternativeNameType {
-        DNS = 0;
-        Email = 1;
+        SUBJECT_ALTERNATIVE_NAME_TYPE_UNSPECIFIED = 0;
+        EMAIL = 1;
         URI = 2;
+        // OID 1.3.6.1.4.1.57264.1.7
```

Comment:
N.B.: Might be worth adding a link to <https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md#1361415726417--othername-san> here.

Reply:
Yeah, good call-out!